app/assets/stylesheets/active_skin.sass

```diff
@@ -31,54 +31,65 @@ body.active_admin
     padding-left: 130px
     color: $skinTextColor
     text-indent: -999999px
+  .header-item
+    top: 0
   a, a:link
     color: $skinTextColor
   ul.tabs
     padding: 0
     font-size: 0
-  ul.tabs > li:hover > a, ul.tabs > li.current > a
-    color: $skinTextActiveColor
-    background: $skinActiveColor
-    @include rounded(0)
-    margin: -10px 0
-  ul.tabs > li 
-    font-size: 12px
-    margin: 0
-    a
-      padding: 13px 20px
+    & > li.menu_item
+      font-size: 12px
       margin: 0
-  ul.tabs > li.has_nested a
-    background-position: 93% 50%
-  ul.tabs > li.has_nested:hover
-    a
-      @include rounded(0)
-      background-color: $skinActiveColor
-      border: none
-      background-position: 93% 50%
-    ul
-      background: $skinActiveColor
-      @include no-shadow()
```

Comment:
This rule is removed to bring back the box shadow from the default theme. This make the cascading menu more visible.

![Screenshot from 2020-04-10 14-39-04](https://user-images.githubusercontent.com/8206622/78974026-3f174d00-7b3b-11ea-8e02-88bc2e02da12.png)